src/Cake.Prca.PullRequests.Tfs/TfsPullRequestSystem.cs

```diff
@@ -303,7 +303,7 @@ private GitHttpClient CreateGitClient()
                 {
                     CommentType = CommentType.System,
                     IsDeleted = false,
-                    Content = issue.Message
+                    Content = ContentProvider.GetContent(issue)
```

Comment:
Does the detection of the threads in the Core Prca still work if we change the message here?